scripts/single-node.sh

```diff
@@ -85,10 +85,14 @@ sed -i'.bak' 's#discard_abci_responses = true#discard_abci_responses = false#g'
 # Override the VotingPeriod from 1 week to 1 minute
 sed -i'.bak' 's#"604800s"#"60s"#g' "${CELESTIA_APP_HOME}"/config/genesis.json
 
+# Override the genesis to use app version 1 and then upgrade to app version 2 later.
+sed -i'.bak' 's#"app_version": "2"#"app_version": "1"#g' "${CELESTIA_APP_HOME}"/config/genesis.json
+
 # Start celestia-app
 echo "Starting celestia-app..."
 celestia-appd start \
   --home "${CELESTIA_APP_HOME}" \
   --api.enable \
   --grpc.enable \
-  --grpc-web.enable
+  --grpc-web.enable \
+  --v2-upgrade-height 3
```

Comment:
Why do we want to have these changes here? is there a reason to upgrade in the `single-node.sh`?

Reply:
Mostly for debugging purposes. IMO this is how arabica, mocha, and mainnet will behave so we should be testing it locally.

It seems non representative of reality to use a genesis file that has app version 2 in it without an upgrade height.